src/chi-vue/src/components/column-customization/ColumnCustomization.tsx

```diff
@@ -96,20 +97,47 @@ export default class ColumnCustomization extends Vue {
     if (this._ColumnCustomizationContentComponent) {
       this._availableColumns = [];
       this._selectedColumns = [];
+      this._selectedData = this.columnsData?.columns.filter((column: DataTableColumn) => column.selected);
       this._processData();
-      (this.$refs.saveButton as HTMLButtonElement).disabled = true;
+      (this.$refs.saveButton as HTMLButtonElement).disabled = false;
```

Comment:
This would leave the save button ENABLED after resetting. What's the logic behind the change?

Reply:
Look the other comment where I explain the correct functionality.